### DISCUSSION TITLE
Does jellyfin support custom repeat intervals?

### DISCUSSION THREAD

**Discussion by mainrs** [jellyfin/jellyfin#11456] [Q&A]
I have some audio files that can be looped, but there loop start- and endpoint do not correspond to the start- and end position of the file itself. I.e., the file is longer than the loop inside of it.

Can jellyfin be told to loop from a given point to another one within a file? I know some audio files support chapters. If jellyfin can loop chapters within a file, that would be enough. I can work with that.

Thank you

**Comment by gnattu** [comment]
Not possible until #10530 is landed. This PR is projected to land in 10.10.

**Reply by majkelwaltuh** [reply]
Is this now possible? #12345 and #12359 have been merged, which seem to be the same thing as #10530.
